artemis/tool.py

```diff
@@ -98,7 +98,7 @@ def call_provision_environment(self, env_name):
         """Do initial provisioning of an environment in Kubernetes and Terraform."""
         self.call_provision_terraform(env_name)
         self.call_provision_kubernetes(env_name)
-        self.call_create_endpoints(env_name)
+        #self.call_create_endpoints(env_name)
```

Comment:
If we don't want to create endpoints automatically when provisioning environment, it's better to remove this completely as opposed to commenting it out.